github-notify.go

```diff
@@ -34,10 +36,22 @@ func retrieveNotifications(client *github.Client) []github.Notification {
 	opts := &github.NotificationListOptions{All: false, Participating: true}
 	var results []github.Notification
 	notes, _, _ := client.Activity.ListNotifications(opts)
+	// Oldest notifications first
+	notes = reverseNotifications(notes)
+
+	// Clean up notRead
+	for i := len(notRead) - 1; i >= 0; i-- {
+		if(!hasNotification(notes, notRead[i])) {
+			notRead = append(notRead[:i], notRead[i+1:]...)
+		}
+	}
+
 	for _, note := range notes {
```

Comment:
Good point, but I would suggest in that case to just send `notes[len(notes)-1]` instead of `notes[0]`. :smile: no reversing at all.